src/main/java/com/xjeffrose/chicago/client/ConnectionPoolManager.java

```diff
@@ -49,14 +54,36 @@ public ConnectionPoolManager(String hostname) {
     connect(new InetSocketAddress(hostname, 12000), listenerMap.get(hostname));
   }
 
+  public void start() {
+    running.set(true);
+  }
+
+  public void stop() {
+    log.info("ConnectionPoolManager stopping");
+    running.set(false);
+    ChannelGroup channelGroup = new DefaultChannelGroup(workerGroup.next());
```

Comment:
Where is this defined **workerGroup**? 
Compilation failure:
[ERROR] 
/src/main/java/com/xjeffrose/chicago/client/ConnectionPoolManager.java:[64,57] cannot find symbol